### PR TITLE
Build C# projects in VS/Rider solution configurations

### PR DESCRIPTION
**Pull request by GoaLitiuM**
Rider's solution wide analysis does not work properly when projects are not included in the active configuration for build.

Also implements experimental MSBuild .targets files for running Flax.Build when building the solution in IDEs, the same way VC++ projects use custom build commands to invoke Flax.Build.

### CODE REVIEW

**Comment by GoaLitiuM** [comment]
Fixes #1651

File: Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs [hunk 1]
```diff
@@ -106,14 +111,14 @@ public override void GenerateProject(Project project, string solutionPath)
             csProjectFileContent.AppendLine("    <LangVersion>11.0</LangVersion>");
             csProjectFileContent.AppendLine("    <FileAlignment>512</FileAlignment>");
 
-            // Needed for Hostfxr
-            csProjectFileContent.AppendLine("    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>");
```
Comment (mafiesto4) [inline Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs:110]: Why was it removed?

File: Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs [hunk 2]
```diff
@@ -106,14 +111,14 @@ public override void GenerateProject(Project project, string solutionPath)
...
-            csProjectFileContent.AppendLine("    <EnableDynamicLoading>true</EnableDynamicLoading>");
```
Comment (mafiesto4) [inline Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs:111]: Why was it removed?

Comment (GoaLitiuM) [reply Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs:111]: These values are not used by IDE, I think these were leftovers from the early prototypes of the .NET 7 support.

**Pull request merged**